src/IB/IBFEMethod.cpp

```diff
@@ -1758,22 +2262,36 @@ IBFEMethod::computeStressNormalization(PetscVector<double>& Phi_vec,
     // Setup extra data needed to compute stresses/forces.
 
     // Extract the FE systems and DOF maps, and setup the FE objects.
-    LinearImplicitSystem& Phi_system = equation_systems->get_system<LinearImplicitSystem>(PHI_SYSTEM_NAME);
+    TransientLinearImplicitSystem& Phi_system =
+        equation_systems->get_system<TransientLinearImplicitSystem>(PHI_SYSTEM_NAME);
     const DofMap& Phi_dof_map = Phi_system.get_dof_map();
     FEDataManager::SystemDofMapCache& Phi_dof_map_cache = *d_fe_data_managers[part]->getDofMapCache(PHI_SYSTEM_NAME);
     std::vector<unsigned int> Phi_dof_indices;
     FEType Phi_fe_type = Phi_dof_map.variable_type(0);
     std::vector<int> Phi_vars(1, 0);
 
+    // things for building RHS of Phi linear system based on poisson solver.
+    const Real cg_penalty = equation_systems->parameters.get<Real>("cg_penalty");
+    const Real ipdg_jump0_penalty = equation_systems->parameters.get<Real>("ipdg_jump0_penalty");
+    const Real ipdg_beta0 = equation_systems->parameters.get<Real>("ipdg_beta0");
+    const std::string Phi_solver = equation_systems->parameters.get<std::string>("Phi_solver");
+    const Real diffusion = equation_systems->parameters.get<Real>("Phi_diffusion");
+    const Real dt = equation_systems->parameters.get<Real>("dt");
```

Comment:
We shouldn't read in `dt` this way.

Reply:
yeah I know... this was a hack.  I'll setup something like you suggested above.